src/wikibase/client.rs

```diff
@@ -11,15 +11,15 @@ pub struct Client {
     token: Option<String>,
 }
 
-pub fn new(config: super::Config) -> Client {
-    Client {
-        client: reqwest::Client::new(),
-        config,
-        token: None,
+impl Client {
+    pub fn new(config: super::Config) -> Self {
```

Comment:
it's really minor, but I'm not a fan of using contextual import, I prefer the full name to `super`